test/types/community/collection/bulkWrite.test-d.ts

```diff
@@ -1,4 +1,4 @@
-import { expectError } from 'tsd';
+import { expectAssignable, expectError } from 'tsd';
```

Comment:
looks like expectAssignable is not used, and this is causing the updated build step to fail